### ISSUE TITLE
Add the range of each independence test statistic to docs

### ISSUE THREAD

**Issue by CaseyWeiner** [neurodata/hyppo#121]
I think `MGC` test statistic is in the interval [-1, 1] and the rest are [0,1]?

**Comment by oakla** [comment]
Would I be correct in thinking this is probably the easiest issue on the list?

**Comment by sampan501** [comment]
yes, minor docstring edits

**Comment by oakla** [comment]
I'm on it. I've created a fork.

If I understand correctly, I'm suppose to link my pull request to this issue once i've finished coding the solution.

**Comment by oakla** [comment]
> I think `MGC` test statistic is in the interval [-1, 1] and the rest are [0,1]?

Can i assume these values are correct, or is it worth taking the time to verify it myself?

**Comment by sampan501** [comment]
Those aren't correct, most are [-1, 1] while some like HHG have no clear interval. I think it's worth taking time to verify, especially if you are learning a lot!

**Comment by Kshitij2001** [comment]
Can you please assign me this issue?

**Comment by oakla** [comment]
I've been working on and off this issue for a while. So I can submit some of my progress, can we break this issue into separate issues for each test?

**Comment by sampan501** [comment]
Yes we can. Can you make a new issue for the tests you have done and then open a PR linking to that issue?

**Comment by oakla** [comment]
I made a PR for one of the tests last week. There's been no comment on it yet

**Comment by sampan501** [comment]
Hi I'm currently out of the country and haven't reviewed any PRs made in the last week. I'll review them when I get back.

**Comment by vikram-8290** [comment]
i want to work on this issue.Can you please assign me this issue?

**Comment by vikram-8290** [comment]
i have made pull request .Please Let me know If have done it correctly or not(bcuz its my first open source issue).

**Comment by sampan501** [comment]
So the pull request was correct, but that part of the documentation was already correct.

For this issue, really we want someone to go through each of the papers and find the range of each test statistic. For example, the RV test statistic has values between -1 and 1.

**Comment by GargRaghav17** [comment]
hey... what papers are you talking about?
\

> So the pull request was correct, but that part of the documentation was already correct.
> 
> For this issue, really we want someone to go through each of the papers and find the range of each test statistic. For example, the RV test statistic has values between -1 and 1.

**Comment by sampan501** [comment]
Each paper has an associated paper that will likely be find in the docs. I believe each of them should have the range specified for the statistic. If you find some that don't, let me know

**Comment by arshm06** [comment]
Hello! I have made a pull request for the RV test statistic. Thank you!

**Comment by oakla** [comment]
> ... really we want someone to go through each of the papers and find the range of each test statistic. 

haha, looking at this thread years after I had a go at this issue. I had been trying to find the ranges by doing a lot of maths.
